src/main/kotlin/no/nav/familie/ef/mottak/service/ArkiveringService.kt

```diff
@@ -24,7 +28,28 @@ class ArkiveringService(private val integrasjonerClient: IntegrasjonerClient,
     fun ferdigstillJournalpost(søknadId: String) {
         val soknad: Soknad = søknadService.get(søknadId)
         val journalpostId: String = soknad.journalpostId ?: error("Søknad mangler journalpostId")
-        integrasjonerClient.ferdigstillJournalpost(journalpostId)
+        val enhet = integrasjonerClient.finnBehandlendeEnhet(soknad.fnr)
+        val journalførendeEnhet = enhet.firstOrNull()?.enhetId
+                                  ?: error("Ingen behandlende enhet funnet for søknad ${soknad.id} ")
+
+        integrasjonerClient.ferdigstillJournalpost(journalpostId, journalførendeEnhet)
+    }
+
+    fun oppdaterJournalpost(søknadId: String) {
+        val soknad: Soknad = søknadService.get(søknadId)
+        val journalpostId: String = soknad.journalpostId ?: error("Søknad mangler journalpostId")
+        val journalpost = integrasjonerClient.hentJournalpost(journalpostId)
+
+        val oppdatertJournalpost = OppdaterJournalpostRequest(
+                bruker = journalpost.bruker?.let {
```

Comment:
Blir det null her ellers? Er det ønskelig?

Reply:
Da var den isåfall "null" fra før - så da er det greit nok.